src/services/etherpad/hooks/Pad.js

```diff
@@ -58,7 +58,7 @@ const before = {
 	find: [disallow()],
 	get: [disallow()],
 	create: [
-		globalHooks.hasPermission('TOOL_CREATE'),
+		globalHooks.hasPermission('TOOL_CREATE_ETHERPAD'),
```

Comment:
it just occured to me that we have also e.g. substitute teacher
a better way would be to keep the TOOL_CREATE check, for teacher, and add with OR condition TOOL_CREATE_ETHERPAD for the student....


```suggestion
		some(globalHooks.hasPermission('TOOL_CREATE'), globalHooks.hasPermission('TOOL_CREATE_ETHERPAD'))
```

Reply:
I have already tried to do this. And it doesn't work. It gives the same error for the student.